javatests/arcs/core/delegates/DelegateTest.kt

```diff
@@ -0,0 +1,39 @@
+package arcs.core.delegates
```

Comment:
I think this file should go in `src/wasm/kotlin/javatests` instead of `javatests/arcs/core/` right?

Reply:
`javatests/arcs/core` will be its eventual home.

However, the file it's actually testing (Entity.kt) is not there currently. I think moving that will be a bigger job, so it's probably better to put it `src/wasm/kotlin/javatests` like you suggest for now...

Reply:
Now at a completely code-generated solution so those tests went away.